superflore/generators/bitbake/run.py

```diff
@@ -54,6 +55,13 @@ def main():
     pr_comment = args.pr_comment
     skip_keys = set(args.skip_keys) if args.skip_keys else set()
     selected_targets = None
+    if not args.dry_run:
+        if 'SUPERFLORE_GITHUB_TOKEN' not in os.environ:
+            raise NoGitHubAuthToken(
```

Comment:
I think this string should be moved to the default constructor of the `NoGitHubAuthToken` (or at least to a common location) since it is duplicated below.